lib/jekyll-github-metadata/value.rb

```diff
@@ -3,6 +3,9 @@
 module Jekyll
   module GitHubMetadata
     class Value
+      extend Forwardable
+      def_delegators :render, :+, :to_s, :to_json, :eql?
```

Comment:
Memoized via 7027532.

> Why `#+`?

Workaround so we don't need to backport https://github.com/jekyll/jekyll/pull/6253.

> Per Object#hash, if you specify #eql? you must also specify #hash:

Done via f0662b0.